src/beautifiers/index.coffee

```diff
@@ -70,6 +70,7 @@ module.exports = class Beautifiers extends EventEmitter
     'yapf'
     'erl_tidy'
     'marko-beautifier'
+    'r-beautifier'
```

Comment:
It should be `formatR` in `src/beautifiers/index.coffee` and rename `src/beautifiers/r-beautifier/index.coffee` to `src/beautifiers/formatR/index.coffee`.